core/src/avm2/globals/flash/ui/ContextMenu.as

```diff
@@ -17,9 +19,15 @@ package flash.ui
         private var _builtInItems: ContextMenuBuiltInItems = new ContextMenuBuiltInItems();
 
         public function get builtInItems(): ContextMenuBuiltInItems {
+            stub_getter("flash.ui.ContextMenu", "builtInItems");
```

Comment:
I don't think you need to add a stub getter since this part is working correctly (returning the value of built-in-items).

Reply:
I actually tried to keep the same format as in other parts of the code like here - https://github.com/ruffle-rs/ruffle/pull/10656/files
but I can remove it if it is useless

Reply:
As done in that PR, the `stub_getter` is unnecessary (that PR doesn't add a `stub_getter`).